apps/wallet-mobile/src/features/Send/useCases/ListAmountsToSend/EditAmount/EditAmountScreen.tsx

```diff
@@ -147,17 +147,16 @@ const AmountInput = ({value, onChange, ticker}: AmountInputProps) => {
       mode="flat"
       autoComplete="off"
       value={value}
-      placeholder={value}
+      placeholder="0"
```

Comment:
the placeholder only is shown when the text is empty. On low-end devices, there is a weird delay when user is typing the amount